apps/rd/rd-judicial-data-load/aat-00.yaml

```diff
@@ -10,3 +10,5 @@ spec:
         DELETE_ORG: false
         JRD_SIDAM_SEARCH_QUERY: "(roles:judiciary) AND lastModified:>now-116h"
         MAIL_FROM: no-reply@mail-rd-nonprod.platform.hmcts.net
+    java:
+      image: hmctspublic.azurecr.io/rd/rd-judicial-data-load:pr-518
```

Comment:
you may not deploy PR images to the AAT environment

Reply:
changed the  PR image pointing to demo environment .